tap_eloqua/utils.py

```diff
@@ -0,0 +1,21 @@
+from typing import Dict
+import json
+from singer import get_logger
+
+LOGGER = get_logger()
+
+def read_config(config_path) -> Dict:
```

Comment:
Add function comments for all functions.

Reply:
done